### DISCUSSION TITLE
When using herd link, could you remove the ".test" if a user incorrectly adds it?

### DISCUSSION THREAD

**Discussion by bdahlinger** [beyondcode/herd-community#909] [Ideas]
New to Herd, running great now, definitly upgrading to Pro...

When I read the docs, I ran `herd link mysite.test`. The docs didn't mention that you don't need the ".test" at the end. As a laravel user, I am conditioned to think it would be what my `APP_URL` would be, so naturally, I added the `.test` to the custom domain name to link. Took me a couple hours to realize this would create a `mysite.test.test` 🤪 

Proposal:

1. Add to documents to not add a ".test" as that will be automatically assumed and added
2. When a new user does append the `.test`, could you remove it? Best of both worlds?

Thanks for the great tool!

(Using Mac and Windows)

<img width="633" alt="doc-example" src="https://github.com/user-attachments/assets/05c03dab-a36b-408c-80c8-985b4653bcbf">

**Comment by sschlein** [comment]
I've created an internal issue for that after running into this problem countless times.
